src/gamecontroller.cs

```diff
@@ -307,16 +353,38 @@ partial interface GCController {
 		[Notification, Field ("GCControllerDidDisconnectNotification")]
 		NSString DidDisconnectNotification { get; }
 
-		[iOS (8,0), Mac (10,10)]
+		[iOS (8,0), Mac (10,10), TV(13,0)]
 		[Export ("motion", ArgumentSemantic.Retain)]
 		GCMotion Motion { get; }
 
 		[iOS (9,0)][Mac (10,11)]
 		[Export ("handlerQueue", ArgumentSemantic.Retain)]
 		DispatchQueue HandlerQueue { get; set; }
+
+		[TV (13, 0), Mac (10, 15, onlyOn64: true), iOS (13, 0)]
+		[Export ("productCategory")]
+		string ProductCategory { get; }
+
+		[TV (13, 0), Mac (10, 15, onlyOn64: true), iOS (13, 0)]
+		[Export ("snapshot")]
+		bool Snapshot { [Bind ("isSnapshot")] get; }
+
+		[TV (13, 0), Mac (10, 15, onlyOn64: true), iOS (13, 0)]
+		[Export ("capture")]
+		GCController Capture { get; }
```

Comment:
That sounds like a method - and not a property
Please check headers and docs (both ObjC and Swift)

Reply:
This one, ControllerWithMicroGamepad, and ControllerWithExtendedGamepad were all methods but sharpie suggested changing them to properties since they just return a GCController, should I leave them as methods?

Reply:
yes, they are not state (but actions) so they should be methods

side note: in _old_ ObjC there was no syntax for property so sharpie does its best... but it's only a suggestion (and newer headers generally use `@property` so there's no/less need to guess properties).

Reply:
Okay thanks!